src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -44,18 +45,20 @@
   private static final String APPLICATION_NAME = "SAY";
```

Comment:
Now that billing is set up can we get our actual API key?

Reply:
```suggestion
  private static final String APPLICATION_NAME = "SAY";I think this is fine for github repo due to the API protection
```